configs/native.yaml

```diff
@@ -2,6 +2,7 @@ plugins:
   - path: ground_truth_slam/
   - path: offline_imu_cam/
   - path: open_vins/
+  - path: pose_lookup/
```

Comment:
Both of these cannot be here. They both register with the same name and cause an assertion failure in phonebook. So we should comment out pose_lookup (uncommon case) and keep pose_prediction (common case).